Flat/src/ModulesLepPho.cc

```diff
@@ -79,11 +79,15 @@ void PandaAnalyzer::SimpleLeptons() {
     panda::Lepton* lep1 = looseLeps[0];
     gt->mT = MT(lep1->pt(),lep1->phi(),gt->pfmet,gt->pfmetphi);
   }
-  panda::Muon *mu1=0, *mu2=0; panda::Electron *ele1=0, *ele2=0;
+  panda::Muon *mu1=0, *mu2=0, *mu3=0, *mu4=0; panda::Electron *ele1=0, *ele2=0, *ele3=0, *ele4=0;
   if (gt->nLooseLep>0) { mu1 = dynamic_cast<panda::Muon*>(looseLeps[0]); ele1 = dynamic_cast<panda::Electron*>(looseLeps[0]); };
   if (gt->nLooseLep>1) { mu2 = dynamic_cast<panda::Muon*>(looseLeps[1]); ele2 = dynamic_cast<panda::Electron*>(looseLeps[1]); };
+  if (gt->nLooseLep>2) { mu3 = dynamic_cast<panda::Muon*>(looseLeps[2]); ele3 = dynamic_cast<panda::Electron*>(looseLeps[2]); };
```

Comment:
Also here... do you intend to use this module for analyses that consider more than 2 leptons?

Reply:
We will need support for 4 or 5 leptons in ZZ selection

Reply:
But you use `complicatedLeptons = True` for that selection. Why complicate an uncomplicated thing?

Reply:
I think this is a question for Dylan, but we need to do something like that (maybe in a less complicated way? not sure)

Reply:
As before, this PR should not touch `SimpleLeptons` I think, because none of the relevant analyses use `SimpleLeptons`.